sphinx/util/osutil.py

```diff
@@ -72,20 +71,25 @@ def ensuredir(file: str | os.PathLike[str]) -> None:
     os.makedirs(file, exist_ok=True)
 
 
-def mtimes_of_files(dirnames: list[str], suffix: str) -> Iterator[float]:
```

Comment:
While those were never documented, to be on the safe side, let's make them follow a deprecation period.

Reply:
There are no uses in the wild: https://github.com/search?q=mtimes_of_files%28+NOT+is%3Aarchived+NOT+is%3Afork+language%3APython&type=code

Reply:
Oh then it's fine I think.